config/checkstyle-non-main-files-suppressions.xml

```diff
@@ -65,6 +65,8 @@
   <suppress id="lineLength"
          files="src[\\/]it[\\/]resources[\\/].*[\\/]InputPackageStatementLongName.*\.java"
          lines="1"/>
+  <suppress id="lineLength"
+         files="src[\\/]it[\\/]resources[\\/].*[\\/]InputNoLineWrapping\.java"/>
```

Comment:
why we have it?
it was not required before.

can we do:
```
 // ok bellow, long imports are allowed
import com.google.checkstyle.test.chapter3filestructure.toolongpackagetotestcoveragegooglesjavastylerule.*;
```

Reply:
for next PRs, lets make comment for each new supprssion.
it is always a red flag for all maintainers.

Reply:
> why we have it?
> it was not required before.

We had InputLineLength.java file to show that long imports are allowed, we have the same name input file in different sections so we already had suppression for it.

In this PR, I combined different input files including InputLineLength.java file of this section, so I had to add LineLength suppression for this file.

>
> can we do:
> ```
>  // ok bellow, long imports are allowed
> import com.google.checkstyle.test.chapter3filestructure.toolongpackagetotestcoveragegooglesjavastylerule.*;
> ```

We need suppression for this file otherwise it'll keep complaining

Reply:
Ok, long line is required , rule https://google.github.io/styleguide/javaguide.html#s3.3.2-import-line-wrapping